spec/safe_migration_spec.rb

```diff
@@ -527,7 +527,7 @@ def up
 
             indexes = ActiveRecord::Base.connection.indexes("foos")
             expect(indexes.size).to eq(1)
-            expect(indexes.first.to_h).to include(:table => "foos", :name => "index_foos_on_bar", :columns => ["bar"])
```

Comment:
Could we break this out as a separate commit?

Reply:
Heh, didn't see this was already merged :D

Reply:
:/ sorry, I probably should have waited